index.html

```diff
@@ -86,201 +86,247 @@ <h1>
+                                            HTML
+                                        </span>
+                                    </li>
+                                    <li class="add-skills">
+                                        <a href="#"><img src="./images//Ellipse 1 (2).png" alt="CSS"></a>
+                                        <span class="text-skills">
+                                            CSS
+                                        </span>
+                                    </li>
+                                </ul>
+                            </li>
+                            <li class="skills">Frameworks <span><img src="./images/Enabled.png" alt="frameworks"></span>
+                            </li>
+                            <li>
+                                <hr class="hr-line">
+                            </li>
+                            <li class="skills">Skills <span><img src="./images/Enabled.png" alt="Skills"></span></li>
+                            <li>
+                                <hr class="hr-line">
                             </li>
                         </ul>
-                    </li>
-                    <li class="skills">Frameworks <span><img src="./images/Enabled.png" alt="frameworks"></span></li>
-                     <li>
-                        <hr class="hr-line">
-                     </li>
-                    <li class="skills">Skills <span><img src="./images/Enabled.png" alt="Skills"></span></li>
-                    <li>
-                        <hr class="hr-line">
-                    </li>
-                </ul>
+                    </div>
+
+
+
+                </section>
+
+                <!-- 
+                contact form -->
+
+                <section class="contact-me">
+                    <img class="contact-bg-image" alt="Image missing">
+                    <div class="contact-form-submit">
+                        <h1>
+                            Contact me
+                        </h1>
+                        <p class="text-form">
+                            If you have an application you are interested in developing, a feature that you need built
+                            or a project that needs coding. I’d love to help with it
+                        </p>
+                        <form action="https://formspree.io/f/mvoyeelg" class="contact-form" method="POST">
+
+                            <input name="name" id="name" type="text" placeholder="Yeremias" maxlength="30" required>
+
+                            <input name="email" type="email" name="replyto" placeholder="natajayanj14@gmail.com"
+                                required>
+
+                            <textarea name="message" cols="30" rows="10" placeholder="Write your message here"
+                                maxlength="500" required></textarea>
+
+                            <button class="submit" type="submit" value="Send"> Get In Touch </button>
```

Comment:
- [ ] The _**get in touch**_ text on this button has a black color on your work but the Figma design has a blue-like color. Please use the right color.